ssp

```diff
@@ -4853,59 +4731,29 @@ sub check_for_cpsources_conf {
                 push @cpsources_servers, $value;
             }
         }
-
-      CPSOURCES_SERVER:
         foreach my $cpsources_server (@cpsources_servers) {
             chomp $cpsources_server;
             my ( $host, $port ) = split( ':', $cpsources_server );
             if ( !defined($port) ) {
                 $port = 80;
             }
-
-            local $SIG{'ALRM'} = sub {
-                print_warn('/etc/cpsources.conf: ');
-                print_warning("Unresponsive mirror: http://$host:$port");
-                next CPSOURCES_SERVER;
-            };
-
-            my $sock = IO::Socket::INET->new(
-                PeerAddr => $host,
-                PeerPort => $port,
-                Proto    => 'tcp',
-                Timeout  => 3,
-            );
-
-            my $tiers_data;
-
-            if ($sock) {
-                print $sock "GET /cpanelsync/TIERS HTTP/1.1\r\nHost: $host\r\n\r\n";
-                sysread $sock, $tiers_data, 1000;
-                close $sock;
-            }
-
-            alarm 0;
-
+            my $tiers_data = _http_get( Host => $host, Port => $port, Path => '/cpanelsync/TIERS', WantHeaders => 1 );
```

Comment:
We ultimately need to handle this differently at some point.  

As of right now TIERS will remain in place - but We should shift emphasis to TIERS.json as there is going to be more object-oriented data in there moving forward.

No date has been set on when it will happen, but TIERS, will be removed at some point in the future though that points is not defined and it will not take place for some time.

Just keep this in mind moving forward.  We can use JSON::XS or something along those lines and actually define full objects which also has it's own advantages.